taskt/Core/Utilities/CommonUtilities/VariableMethods.cs

```diff
@@ -286,16 +286,21 @@ public static string ConvertToUserVariable(this String str, object sender)
                 }
 
                 //test if math is required
-                try
+                if (mathChars.Any(f => str.Contains(f)))
```

Comment:
What If `str` consists of a date string like "07-21-2020"?
It will be considered a math expression.

Reply:
honestly I don't know. I wasn't trying to modify the code and its functionality, I was just getting rid of the try/catch because it was filling my Visual Studio output with tons of caught exception messages. I believe if you look a couple of lines above that, there's a date format check though.

Reply:
Basically it was using the try/catch as a logic checker and every time it was "false", it would catch the exception and continue.

Reply:
Got it